lisa/sut_orchestrator/hyperv/source.py

```diff
@@ -1,25 +1,23 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT license.
 from pathlib import PurePath
-from typing import Any, List, Type
+from typing import List, Type
 
 from lisa import schema
 from lisa.node import RemoteNode
 from lisa.tools.ls import Ls
 from lisa.tools.remote_copy import RemoteCopy
 from lisa.tools.unzip import Unzip
 from lisa.util import InitializableMixin, subclasses
-from lisa.util.logger import get_logger
+from lisa.util.logger import Logger, get_logger
 
 from .schema import LocalSourceSchema, SourceFileSchema, SourceSchema
 
 
 class Source(subclasses.BaseClassWithRunbookMixin, InitializableMixin):
-    def __init__(self, runbook: SourceSchema, **kwargs: Any) -> None:
+    def __init__(self, runbook: SourceSchema, parent_logger: Logger) -> None:
```

Comment:
Keep `**kwargs: Any` here and when calling super(). So if other functionality implemented in base classes, it won't break.

Reply:
Added. It is also worth noting that the `Source` in baremetal platform doesn't have `kwargs` and someone needs to add it there too.

Reply:
@LiliDeng Please add kwargs in BareMetal Typed schemas.